components/play-console/element.css

```diff
@@ -29,7 +29,12 @@ li {
   }
 }
 
-code {
+pre {
+  display: inline-block;
+
+  margin: 0;
```

Comment:
Unfortunately, this doesn't look good with messages spanning multiple lines:

<img width="782" height="460" alt="image" src="https://github.com/user-attachments/assets/748afb15-11ba-48ce-863b-bd95d92b0394" />

To avoid this, we could set:

```css
  display: inline;
  white-space: preserve;
```

<img width="782" height="142" alt="image" src="https://github.com/user-attachments/assets/31e2effd-bb1d-4b5a-986a-61bda1341491" />

And if we want to keep the indent for subsequent lines, we also need:

```css
li {
  display: flex;
  gap: 0.5ch;
}
```

Reply:
I tried solution with `li::marker` but Safari hasn't implemented the marker with content.
We don't need  flex if we  use `vertical-align` on ::before. Check the latest commit.